src/Xbehave.2.Core/ScenarioAttribute.cs

```diff
@@ -19,7 +19,11 @@ namespace Xbehave
     /// E.g. <see cref="Xunit.InlineDataAttribute"/> or
     /// <see cref="Xunit.MemberDataAttribute"/>.
     /// </summary>
+#if PLATFORM_DNX
+    [XunitTestCaseDiscoverer("Xbehave.Execution.ScenarioDiscoverer", "xunit.execution.xbehave")]
+#else
     [XunitTestCaseDiscoverer("Xbehave.Execution.ScenarioDiscoverer", "Xbehave.Execution.{Platform}")]

```

Comment:
The platform should always be `"Xbehave.Execution.{Platform}"`. I.e. the assembly for the `dotnet` TFN should be named `"Xbehave.Execution.dotnet"`.